src/devices/Gpio/Drivers/OrangePiZero2Driver.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.Gpio.Drivers
+{
+    /// <summary>
+    /// A GPIO driver for the Orange Pi Zero 2.
+    /// </summary>
+    /// <remarks>
+    /// SoC: Allwinner H616 (sun50iw9p1)
+    /// </remarks>
+    public class OrangePiZero2Driver : Sun50iw9p1Driver
+    {
+        public static readonly int[] _pinNumberConverter = new int[]
```

Comment:
This generates a "Missing XML Documentation" error. Probably it shouldn't be public?